README.md

```diff
@@ -1 +1,57 @@
-# angular-template-readme
\ No newline at end of file
+# Template and templateUrl
+
+## Overview
+
+We've created a directive with a template - let's take a deeper look into that.
+
+## Objectives
+
+- Describe template and templateUrl
+- Write a template using "template"
+- Write a template using "templateUrl"
+
+## Templates
+
+You might've noticed that we've created a template item in our directive's object.
+
+This will be put into the DOM where the directive is invoked.
+
+Templates can only contain one root element! If we use more, Angular will error because it can't keep track of the elements properly.
+
+```js
+function MyDirective() {
+	return {
+		template: '<div>Hello world!</div>'
+	};
+}
+
+angular
+	.module('app')
+	.directive('myDirective', MyDirective);
+```
+
+## templateUrl
```

Comment:
Before this section, show the problem of not using a separate template by writing out a longer template string. This will show the need to move that into a separate file.